indexer/src/types/transaction.ts

```diff
@@ -93,7 +94,7 @@ export function toEthTx({
     input: txJSON.data!,
     nonce: txJSON.nonce!,
     to: transaction.to?.toString() ?? null,
-    transactionIndex: isPendingBlock ? null : padBigint(BigInt(index ?? 0), 32),
+    transactionIndex: padBigint(BigInt(index ?? 0), 32),
```

Comment:
As transaction index is a `u64`, shouldn't we pad this with 16 only and not 32 in order to avoid confusion with hashes as we did with block numbers?

Reply:
cc @greged93

Reply:
can do yes

Reply:
will this change anything on RPC side?

Reply:
so pad 8 right?

Reply:
I don't think so, should be ok, we did the same on block number. Worst case we will see it in the CI :)

Reply:
Sorry, mb, you need to update `transaction_by_block_hash_and_index` and `transaction_by_block_number_and_index` to use `U64_PADDING`